python/common/thief_constants.py

```diff
@@ -3,7 +3,91 @@
 # full license information.
 
 
-class ServiceAckType(object):
-    TELEMETRY_SERVICE_ACK = "telemetry"
-    ADD_REPORTED_PROPERTY_SERVICE_ACK = "add_reported"
-    REMOVE_REPORTED_PROPERTY_SERVICE_ACK = "remove_reported"
+class Const(object):
+    JSON_CONTENT_TYPE = "application/json"
+    JSON_CONTENT_ENCODING = "utf-8"
+    JSON_TYPE_AND_ENCODING = {
+        "contentType": "application/json",
+        "contentEncoding": "utf-8",
+    }
+    PROPERTIES = "properties"
+    REPORTED = "reported"
+
+
+class Fields(object):
+    class Deprecated(object):
```

Comment:
>Deprecated [](start = 10, length = 10)

I'm going to remove the functionality around these strings, but I wanted to do that as a separate PR.